package/Dockerfile.lite

```diff
@@ -68,12 +68,15 @@ RUN python3 -m venv /var/lib/python-venv \
     && poetry export --format requirements.txt | /var/lib/python-venv/bin/pip3 --no-cache-dir install -r /dev/stdin \
     && apk del build-base python3-dev libffi-dev
 
-COPY package/etc/syslog-ng.conf /etc/syslog-ng/syslog-ng.conf
+COPY package/lite/etc/syslog-ng.conf.jinja /etc/syslog-ng/syslog-ng.conf.jinja
 
-COPY package/etc/conf.d/conflib/fallback /etc/syslog-ng/conf.d/conflib/fallback
-COPY package/etc/conf.d/conflib/raw /etc/syslog-ng/conf.d/conflib/raw
```

Comment:
why do we remove copying fallback and raw?

Reply:
because for `lite` we have separate directory `conf.d/conflib` that we copying on one operation